src/views/TreasuryDashboard/components/Graph/Graph.js

```diff
@@ -1,6 +1,6 @@
 import { t } from "@lingui/macro";
 import { useTheme } from "@material-ui/core/styles";
-import Chart from "src/components/Chart/Chart.jsx";
+import Chart from "src/components/Chart/Chart.tsx";
```

Comment:
Nit: can we remove the extension completely i.e. to the following?

```
import Chart from "src/components/Chart/Chart";
```

Reply:
Yes